app/models/agent.rb

```diff
@@ -26,6 +26,9 @@ class Agent < ApplicationRecord
   scope :active, -> { where(deleted_at: nil) }
   scope :order_by_last_name, -> { order(Arel.sql('LOWER(last_name)')) }
   scope :secretariat, -> { joins(:service).where(services: { name: 'Secrétariat'.freeze }) }
+  scope :available_for_rdv, lambda { |motif, service|
+    motif.for_secretariat ? joins(:service).where(service: service).or(secretariat) : where(service: service)
```

Comment:
I don't think the joins is necessary in the first part of the condition, is it?

Reply:
without I can't do .or() because both parts must be identical

Reply:
oh right I just tested I didn't know that; sorry.

slightly unrelated / larger than your PR but I think it'd be nice to add this to the Service model:

```
def self.secretariat
    find_by!(name: SECRETARIAT)
end
```

so we could rewrite these agents' scopes as: 

```
scope :secretariat, -> { where(service: Service.secretariat) } 
scope :available_for_rdv, lambda { |motif| 
  mot.for_secretariat ? where(service: [motif.service, Service.secretariat]) : where(service: motif.service)
}
```